legacy_packages/wallets/src/evm/connectors/blocto/index.ts

```diff
@@ -145,7 +145,7 @@ export class BloctoConnector extends WagmiConnector<
   }
 
   async isAuthorized(): Promise<boolean> {
-    return !!this._provider?._blocto?.sessionKey ?? false;
+    return !!this._provider?._blocto?.sessionKeyEnv ?? false;
```

Comment:
this is unlikely to be accurate - couldn't quite find where "sessionKey" is used in the upgraded blocto package